src/App.tsx

```diff
@@ -1,61 +1,124 @@
 import './App.scss';
+import { useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { TodosList } from './types/TodoList';
+
+const maxId = (...todos: TodosList[]) => {
+  const getAllIds = todos.map(todo => {
+    return todo.id;
+  });
+
+  return Math.max(...getAllIds);
+};
+
+export const App: React.FC = () => {
+  const [todo, setToDo] = useState('');
+  const [selectedNameId, setSelectedNameId] = useState(0);
+  const [todos, setTodoList] = useState([...todosFromServer]);
+  const [isTitleEmpty, setIsTitleEmpty] = useState(false);
+  const [isSelectEmpty, setIsSelectEmpty] = useState(false);
+  const [todoId, setTodoId] = useState(maxId(...todos));
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setToDo(
+      event.target.value.replace(/[^a-zA-Za-åa-ö-w-я 0-9_]/gi, ''),
+    );
+
+    if (event.target.value.length > 0) {
+      setIsTitleEmpty(false);
+    } else {
+      setIsTitleEmpty(true);
+    }
```

Comment:
error should appear only on submit
![image](https://user-images.githubusercontent.com/37161315/205609994-c8d31c67-01ed-4a68-ac63-0e415b4857d9.png)
```suggestion
```